resources/resource_annex_change_history.adoc

```diff
@@ -15,10 +15,10 @@ Unless otherwise specified all modifications are upwardly compatible to the prev
 
 * the mapping tables of ISO 10303 application protocols based on the previous edition of this document remain valid in a revision of that application protocol based on this edition of this document.
 
-[yaml2text,schemas.yaml,context]
+[data2text,context=schemas.yaml]
 ------
 {% liquid
-assign path_prefix = "../../schemas/resources/"
+assign path_prefix = "schemas/resources/"
```

Comment:
@kwkwan can we fix data2text to respect the location of the file path?

Reply:
@ronaldtse yes, that would be really useful. For module documents, I need to have access to a `changes.yaml` file that is placed at the same location as `document.adoc`.

Maybe we could do something like this for `loadfile` macro?

```
// fetch respect the document location (e.g. document.adoc)
{% assign data = path | loadfile %}

// fetch respect the root folder where command is executed
{% assign data = path | loadfile: "." %}
```

Reply:
@ronaldtse I think `data2text` is indeed fixed to the file location (which in this case is `documents/iso-10303-*/` folder).
The reason I redefined the indicated string is because of `loadfile: "."` macro. Given that it is taking the root folder (where command is executed) as reference. Hence, the change.

However, for the `modules/module_annex_change_history.adoc` template, I need to have access to a `changes.yaml` file that is placed at the same location as `document.adoc`.

Maybe we could do something like this for `loadfile` macro?

```
// fetch respect the document location (e.g. document.adoc)
{% assign data = path | loadfile %}

// fetch respect the root folder where command is executed
{% assign data = path | loadfile: "." %}
```